### ISSUE TITLE
Bug: Player model clips into floor when crouch jumping

### ISSUE THREAD

**Issue by Equalizer5118** [StrataSource/Portal-2-Community-Edition#1234]
### Describe the bug

When couch jumping with mv_duckjump enabled, the playermodel will clip into the floor. This only happens when crouchjumping with crouch then jump, and not jump then crouch. Can be seen both in thirdperson and in mirrors

### Issue Map

N/A, easily tested here: https://www.mediafire.com/file/rq674gsa7ptjehq/mirrortest.bsp/file
(simple map I created to test prop_mirror. saves having to use sv_cheats and enabling third person)

### To Reproduce

1. Open any map
2. Enable mv_duckjump
3. duckjump, but press crouch first
4. Observe bug

### Operating System

Tested on Windows 11

**Comment by JJL772** [comment]
We're going to be removing the dynamic movement stuff (all `mv_` cvars)

**Comment by JJL772** [comment]
well all of that code is going to be removed in favor of a different system

**Comment by MyGamepedia** [comment]
Maybe stupid question but: does it mean that old movement (as in Portal/Half Life 2) will not added ? I'm pretty sure that it'll can call problems like this.

**Comment by JJL772** [comment]
Yes, HL2 movement will be added. This isn't that though, this was a sloppy attempt at porting HL2 movement features to Portal 2's movement

**Comment by MyGamepedia** [comment]
Ok. By the way, how hard will be to add it ?
